ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/datepicker.js

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+use(function () {
```

Comment:
Looks like this would be same for all fields, @barshat7 can we move this to base, so that other components could re-use it ?

Reply:
common code can be moved in future iterations.

Reply:
customizations are intuitive if this is within the component. Do you see a use case where same renderer customisations are intended to be applied across components?